### DISCUSSION TITLE
Is there a wrapper around lexical to make things easier?

### DISCUSSION THREAD

**Discussion by amirhhashemi** [facebook/lexical#2419] [Q&A] [answered]
So I have to write 700 lines of code to add a [simple toolbar](https://codesandbox.io/s/lexical-rich-text-example-5tncvy?file=/src/plugins/ToolbarPlugin.js:335-351). No no no, I can't maintain that. 
Is there some kind of wrapper around lexical to make things easier and more maintainable? kind of like what [tiptap](https://tiptap.dev/) does for [ProseMirror](https://prosemirror.net/).

**Comment by thegreatercurve** [comment, marked as answer]
Not yet, as Lexical as only been open sourced fairly recently. The Meta team have no plans to release any such wrapper externally, as all of our internal integrations are heavily customised, but the project/business idea is there though if someone else wants to pick it up.

**Reply by armenic** [reply]
I guess this is why I can not find a single YouTube demo about lexical 😄 The first demo on YouTube is going to quickly become viral since there is no competition.

**Reply by armenic** [reply]
Here we go the first video https://youtu.be/dx7HSB4Q-I8 and the author seems to be excited. Not quite tutorial though. Really looking forward to video tutorials 👐

**Reply by acywatson** [reply]
Someone recently made Verbum: https://github.com/ozanyurtsever/verbum

I can't attest to the quality of this yet (haven't tried it), but maybe one of the [contributors](https://github.com/ozanyurtsever) can help answer any questions.
